test/sagemaker_tests/tensorflow/tensorflow2_training/integration/local/test_training.py

```diff
@@ -51,7 +51,7 @@ def test_py_versions(sagemaker_local_session, docker_image, py_full_version, fra
         tar.extractall(path=str(tmpdir), members=[output_file])
 
     with open(os.path.join(str(tmpdir), 'py_version')) as f:
-        assert f.read().strip() == py_full_version
+        assert f.read().strip()[0] == py_full_version[0]
```

Comment:
Why only compare the first character of the string?

Reply:
This is no longer needed now.